scripts/gitlab/publish-awss3.sh

```diff
@@ -40,10 +40,15 @@ done
 echo "__________Push binaries to AWS S3____________"
 aws configure set aws_access_key_id $s3_key
 aws configure set aws_secret_access_key $s3_secret
-if [[ "$CI_COMMIT_REF_NAME" = "beta" || "$CI_COMMIT_REF_NAME" = "stable" || "$CI_COMMIT_REF_NAME" = "nightly" ]];
-  then
+
+case "${SCHEDULE_TAG:-${CI_COMMIT_REF_NAME}}" in
+  (beta|stable|nightly)
```

Comment:
is that even possible?

Reply:
for sure, tested :)
the value of the case statement is evaluated before matching the cases. but we can also keep the 3-fold if conditional if you like.